.editorconfig

```diff
@@ -0,0 +1,28 @@
+# http://editorconfig.org
+
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.go]
+indent_style = tab
+indent_size = 8
+
+[*.tmpl]
```

Comment:
Could you remove unused sections ? This repos will never contains `tmpl` `less` `yml` or `js` files